### ISSUE TITLE
Bug: Choose wrong default value if attribute does not exist

### ISSUE THREAD

**Issue by VovaStelmashchuk** [growthbook/growthbook-kotlin#16]
The SDK get value from condition if condition field does not exist into attributes. 
Growthbook response:
```json
{
 "defaultValue": "Default value",
 "rules": [
   {
     "condition": {
       "country": "IN"
     },
     "force": "Default value for country:IN"
   },
   {
     "condition": {
       "brand": "KZ"
     },
     "force": "Default value for brand:KZ"
  }
 ]
}
```

User attributes:
```json
{"brand":"KZ"}
```

Expected result 
Feature value is `Default value for brand:KZ`
Actual result
Feature value is `Default value for country:IN`

Can you approve it is a bug ?

**Comment by jdorn** [comment]
Yes, this looks like a bug.  There is a test case for something similar already - https://github.com/growthbook/growthbook-kotlin/blob/main/GrowthBook/src/commonTest/kotlin/com/sdk/growthbook/tests/GBTestCases.kt#L394

Can you add an additional test case that reproduces your issue?

**Comment by VovaStelmashchuk** [comment]
Yep, I will  create a PR with my unit test.

**Comment by VovaStelmashchuk** [comment]
I fix the issue into #17 .

**Comment by vazarkevych** [comment]
Closed the issue as it is already resolved and merged in #17.
